src/main/java/com/javadiscord/jdi/internal/models/guild/WidgetObject.java

```diff
@@ -0,0 +1,15 @@
+package com.javadiscord.jdi.internal.models.guild;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record WidgetObject(
+    @JsonProperty("id") long id,
+    @JsonProperty("name") String name,
+    @JsonProperty("instant_invite") String instantInvite,
+    // @JsonProperty("channels") List<PartialChannel> channels, // TODO: future
```

Comment:
Why are these TODOs and commented out?

Reply:
It was before we had discussed on discord how to do partial channels, I'll fix that!